test/ng/directive/inputSpec.js

```diff
@@ -4211,6 +4211,22 @@ describe('input', function() {
       expect(inputElm[0].getAttribute('value')).toBe('something');
     });
 
+    it('should update the dom "value" to "" and attribute to null when the binding is set to undefined', function() {
```

Comment:
The reason why I took this was because this test has a comparable way of describing itself: 
https://github.com/angular/angular.js/blob/master/test/ng/directive/inputSpec.js#L4205

So combining my solution with yours, I'd propose: 

`Should clear the "dom" value and attribute when the value is undefined`.

Yet I do think this being somehow odd, talking about clearing both feels like having the same result for both. However, dom value is set to `''` whereas attr is set to `null`.